examples/redis-backed-cache-testng/build.gradle

```diff
@@ -0,0 +1,13 @@
+dependencies {
+    compile 'redis.clients:jedis:3.0.1'
```

Comment:
Should use the "new" Gradle configurations of the Java plugin e.g. `implementation` etc. Same for all the other example projects.

Reply:
If I recall correctly, @bsideup had some kind of antipathy against them :smile: 
But yes, we should have a consistent style and going further the new configs are the way to go I assume.

Reply:
The "old" configurations like `compile` and friends have been soft-deprecated. IMHO Gradle users don't know what that means but essentially this means that the "new" ones should be preferred. The plan is for the old ones to go away.